employees/templates/employees/author_report_list.html

```diff
@@ -18,11 +18,17 @@ <h1>{{ object.email }} {{ title_date }}{{ UI_text.PAGE_TITLE.value }}</h1>
 <a href="{% url 'custom-users-list' %}" class="btn btn-primary hidden-print">
     {{ UI_text.RETURN_BUTTON_MESSAGE.value }}
     {% with date=path|extract_year_and_month_from_url %}
-        <a href="{% url 'export-data-xlsx' pk=object.id year=date.0 month=date.1 %}" class="btn btn-info">
+        <a href="
+            {% if not only_projects_reports %}
```

Comment:
After you do https://github.com/Code-Poets/sheetstorm/pull/356/files#r298811905 you have all the necessary data to add this logic directly to `AuthorReportView`.

Reply:
ok

Reply:
There is no closing `">`.

Reply:
Done

Reply:
You have missed `"`.

Reply:
I can't believe that I could forgot this ... I am so sorry ... and done